packages/core/src/transform-structure-to-ts.test.ts

```diff
@@ -144,7 +144,11 @@ describe('transformStructureToTs', () => {
           | string
           | number
           | {
+              _key: string;
+              _type: "objectLike";
               properties?: {
+                _key: string;
+                _type: "propertyPair";
```

Comment:
@ricokahler the tests failed here because these properties were added. I think it makes sense but I'm not fully sure these properties are there in this case?